HelpSource/Classes/SoundFileView.schelp

```diff
@@ -213,16 +213,26 @@ METHOD:: zoomSelection
     argument::
         The index of the selection; an Integer between 0 an 63.
 
+METHOD:: xZoom
+    The number of seconds of audio to display in the view. E.g., to zoom out by a factor of 2, code::view.xZoom = dataDuration / 2::. (You are responsible for keeping track of the data duration.)
+
+    returns:: A Float, in seconds of audio displayed.
+
+METHOD:: yZoom
+    Vertical scaling. code::yZoom = 1:: sets 0 dBFS to the top and bottom of the view. 0.5 is half as tall.
```

Comment:
> 0.5 is half as tall.

This could be improved. What dBFS do yo get when setting `yZoom` to 0.5?

Reply:
If dBFS are too confusing, it's probably better to remove the reference and say +/-1 instead.

If the audio file is properly engineered, you won't have positive dBFS at all (and, it's literally impossible to have positive dBFS in an integer-format audio file, because the maximum possible integer value for the bit depth maps onto 0 dBFS -- you might have values outside +/-1.0 in a floating-point file but it's not recommended to rely on that). So the question here doesn't entirely make sense.

Or should I use yZoom = 2 instead? Avoid the whole topic of out-of-range values.

Reply:
I think you should stick with +/-1 instead of dBFS, and say that `yZoom=2` will give you +/-0.5.

Reply:
That works for me.

As an editorial aside, I find this last sort of comment -- a concrete suggestion for improvement -- to be *very* helpful. Otherwise, the PR author is left to guess what the desired improvement is, which only wastes everybody's time. (TBH I didn't know exactly what to do with the question about dBFS.)

Reply:
You're right. I'll try to offer more concrete suggestions in the future.